enigma-core/app/src/db/dal.rs

```diff
@@ -15,6 +15,8 @@ pub struct DB {
     // the DB needs to store the options for creating new
     // cf's that would be able to imitate the DB behaviour
     pub options: Options,
+    // keeps track of the status of the state
+    is_stateful: bool,
```

Comment:
I think that this name `is_stateful` and the comment do not express the meaning of the attribute.

Reply:
will change it. `state_updated` is sufficient?